apps/sasquatch/src/main/java/com/microsoft/appcenter/sasquatch/activities/SettingsActivity.java

```diff
@@ -73,6 +73,12 @@
 
     private static boolean sAnalyticsPaused;
 
+    private static String initialStartType;
+
+    private static String oneCollectorLogUrl = "https://mobile.events.data.microsoft.com/OneCollector/1.0";
+
+    private static String otherLogUrl = "https://in-integration.dev.avalanch.es";
```

Comment:
This value would be different based on build variant. If the app is built with jcenterDependency then the logUrl should be pointing to PROD ingestion.
You should be able to find URLs already defined in `env.xml`. Just read the URL by `log_url` key. It will automatically pick up a right value based on current build variant.

Reply:
Yes, I changed it in env.xml.